doc/userguide/src/ExtendingRobotFramework/CreatingTestLibraries.rst

```diff
@@ -1189,7 +1189,19 @@ messages, specify the log level explicitly by embedding the level into
 the message in the format `*LEVEL* Actual log message`, where
 `*LEVEL*` must be in the beginning of a line and `LEVEL` is
 one of the available logging levels `TRACE`, `DEBUG`,
-`INFO`, `WARN`, `FAIL` and `HTML`.
+`INFO`, `WARN`, `FAIL`, `ERROR` and `HTML`.
+
+
+Errors
+''''''''
+
+Messages with `ERROR` level are automatically written into `the
+console and into separate Test Execution Errors section`__ in log
+files. This makes errors more visible than other messages and allows
+using them for reporting important but non-critical problems to users.
+
+__ `Errors and warnings during execution`_
+
```

Comment:
This is identical documentation as with `Warnings` below, right? Why not just have one section `Warnings and errors` instead? Should also add a note that with errors this functionality is new in 2.9.